packages/trader/src/Modules/SmartChart/Components/Markers/accumulators-chart-elements.tsx

```diff
@@ -1,16 +1,29 @@
 import { filterByContractType } from 'App/Components/Elements/PositionsDrawer/helpers/positions-helper.js';
-import PropTypes from 'prop-types';
 import React from 'react';
-import AccumulatorsProfitLossTooltip from './accumulators-profit-loss-tooltip.jsx';
+import AccumulatorsProfitLossTooltip from './accumulators-profit-loss-tooltip';
 import CurrentSpotEmphasizer from './current-spot-emphasizer.jsx';
+import { ProposalOpenContract } from '@deriv/api-types';
+
+type TAccuContractType = {
+    contract_info: Omit<React.ComponentProps<typeof AccumulatorsProfitLossTooltip>, 'className' | 'alignment'> &
+        Required<Pick<ProposalOpenContract, 'underlying' | 'shortcode' | 'contract_id' | 'contract_type'>>;
+};
+
+type TAccumulatorsChartElements = {
+    all_positions: TAccuContractType[];
```

Comment:
@henry-deriv let's rename it because all_positions passed here include not only accumulator positions but all ongoing positions of any types 🙂```suggestion
    all_positions: TPosition[];
```